README.md

```diff
@@ -9,6 +9,12 @@ A home assistant integration for controlling Jci Hitachi devices, using [LibJciH
 
 ## Installation
 
+### With HACS (Recommended)
+1. Install [HACS](https://hacs.xyz/docs/installation/installation).
```

Comment:
@aitjcize this is a 404.

Reply:
Sent: https://github.com/qqaatw/JciHitachiHA/pull/111